week-2/F-logical-operators/exercise.js

```diff
@@ -13,24 +13,25 @@ var cssLevel = 4;
 // (hint: use the comparison operator from before)
 var htmlLevelAbove5;
 var cssLevelAbove5;
+htmlLevelAbove5 = htmlLevel > 5;
+cssLevelAbove5 = cssLevel == 5;
```

Comment:
This isn't quite right - this checks if `cssLevel` is exactly 5 not whether it is above 5. How could we fix this?